opentelekomcloud/validators.go

```diff
@@ -88,3 +88,13 @@ func validateJsonString(v interface{}, k string) (ws []string, errors []error) {
 	}
 	return
 }
+
+func validateKmsKeyStatus(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
you could probably use the built-in terraform validation method [`StringInSlice`](https://github.com/hashicorp/terraform/blob/master/helper/validation/validation.go#L75) for this.

Reply:
Done